kafka_topic_enforcer/src/main/java/com/tesla/data/topic/enforcer/DumpCommand.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2019. Tesla Motors, Inc. All rights reserved.
```

Comment:
We haven't been using copyright some in other files, instead just having the license stand in for permissions/ownership.

Reply:
All java source files have it, so added for consistency. Did you mean non java files? We can do a clean sweep in a separate commit?

Reply:
sg